### DISCUSSION TITLE
This browser does not run on windows 7 32 bit ?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#1757] [Q&A] [answered]
This browser does not run on windows 7 32 bit ?

**Comment by Syndamia** [comment, marked as answer]
Yes, Chromium dropped 32bit support and Electron is getting there. This browser is "based" on those two things, so currently, we have also dropped 32bit support (although it might still be kinda possible?).

Duplicate of/Refer to #1369

**Reply by ghost** [reply]
Thanks though I can't use it : )

**Reply by Syndamia** [reply]
Quoting the linked issue:
> I think the last supported version was 1.9.2, but I also believe Firefox still supports 32-bit, so you're better off using that if you can get the latest version.

**Reply by ghost** [reply]
Thanks, I don't use Firefox cause it consume a lot of resource of my computer

**Reply by ghost** [reply]
uh.. I can't find the 1.9.2 version of min : )

**Reply by PalmerAL** [reply]
For Windows 32-bit, it looks like up to 1.16 would work: https://github.com/minbrowser/min/releases/tag/v1.16.1, or maybe a newer one; I've sort of forgotten at this point. (1.9.2 is the latest for Linux 32-bit).

I wouldn't recomment using that though; that version is almost a year old and has known security issues. If you use Firefox, you'll still get updates, which will be a lot more secure.
